src/handlers/dashboard-extract/handler.ts

```diff
@@ -0,0 +1,27 @@
+import { Athena } from "aws-sdk/clients/all";
+
+import { ResultSet } from "@aws-sdk/client-athena";
+import { putTextS3 } from "../../shared/utils";
+import { AthenaQueryExecutor } from "./athena-query-executor";
+import { ExtractDataReformatter } from "./extract-data-reformatter";
+
+const EXTRACT_KEY = "btm_extract_data/full-extract.json";
+
+export const handler = async (): Promise<void> => {
```

Comment:
You could use the context here, shouldn't be too big of a change

Reply:
This change is in now.